core/pom.xml

```diff
@@ -47,6 +47,11 @@
       <version>1.7.0</version>
     </dependency>
     <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.5</version>
```

Comment:
Jenkins widely uses 3.x: https://github.com/search?l=Maven+POM&p=2&q=org%3Ajenkinsci+%22commons-lang3%22&type=Code . The dependency should be fine